src/components/Person.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {name}
```

Comment:
Use interpolation, it'll make code more readable
```suggestion
      {`My name is ${name}`}
```